config/datasets/user-activity.json

```diff
@@ -48,8 +47,8 @@
             "avg_intensity": {
                 "title": "Average Intensity",
                 "description": [
-                    "Average Intensity measures the engagement level of a typical Firefox Desktop client. It is the proportion of time that a client actively engaged with the browser.",
-                    "Overall, the typical Firefox client engages with the browser about a third of the time when the browser is open. European countries seem to have the highest levels of engagement, with France, Poland, and Italy all showing Average Intensity over 0.35 and Germany approaching 0.40. Most other countries in the top 10 have an Average Intensity around 0.30."
+                    "Intuitively, intensity shows how many days per week do users use the browser.",
```

Comment:
I think there's a typo in this line. It looks like it should be:

> Intuitively, intensity shows how many days per week users use the browser.

Reply:
I think there might be a better way to rephrase this one without sacrificing accuracy, but it's a tricky one. I'll ping Su for his thoughts. :smiley:

Reply:
Yes, this one also tricky because the way this metric is calculated changed. Right now it is the same "intensity" we have in GUD and other places.

Reply:
Hey, did we change the definition of intensity? I thought it was originally defined as active hours / session length, so it's how actively they were interacting with the browser while they had it open. I don't think it's the same as days per week unless the metric def has changed)